client/rechallenge/transformers.js

```diff
@@ -26,6 +26,13 @@ const transformersForHtmlJS = {
       transformer: function addLoopProtect(file) {
         return updateContents(loopProtect(file.contents), file);
       }
+    },
+    {
+      name: 'replace-nbsp',
+      transformer: function replaceNBSP(file) {
+        const nbspRegExp = new RegExp(String.fromCharCode(160), 'g');
```

Comment:
Needs to be a RegExp to ensure all `&nbsp;`'s are removed

From [w3schools](http://www.w3schools.com/jsref/jsref_replace.asp)

> If you are replacing a value (and not a regular expression), only the first instance of the value will be replaced.